accounting-service/app/src/main/java/com/microservices/accounting/temporaryclasses/CardInfo.java

```diff
@@ -0,0 +1,7 @@
+package com.microservices.accounting.temporaryclasses;
```

Comment:
What is this? temporary classes? Why should it be imported from outside?

Reply:
The exact same class is in the user-management service. If you want, I can delete all comments and it would be just a duplicate of com.microservices.usermanagement.entity.CardInfo

Reply:
delete this comments and package please